### PR TITLE
Change color scheme and font, add content to education and skills and experiences sections

### PR DESCRIPTION
**Pull request by IshaniSanturkar**
In this pull request I:
- changed the overall color scheme to teal and white
- separated the original sections that were all on one page onto separate linked HTML pages
- added content, images and formatting to the skills and experiences and education section
- moved all styling information from .html to .css files
The portfolio now looks like this: [portfoliov3.zip](https://github.com/IshaniSanturkar/step/files/4698359/portfoliov3.zip)

### CODE REVIEW

File: portfolio/src/main/webapp/education.html [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  <head>
+    <meta charset="UTF-8">
+    <title>Ishani Santurkar's Portfolio</title>
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body id="content">
+    <header>
+        <a class="headerelem" href="index.html">About Me</a>
```
Comment (fenghaolw) [inline portfolio/src/main/webapp/education.html]: nit: use 2-space indentation 
https://google.github.io/styleguide/htmlcssguide.html#Indentation

Unfortunately cloud shell does not auto-format HTML correctly, so you either need to manually format (which is boring) or set up a local IDE (which might be useful for later). VSCode is probably a good choice if you choose the later. Let me know if you need help for this!

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -1,11 +1,16 @@
 body {
-    font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+   font-family: 'Montserrat', sans-serif;
```
Comment (fenghaolw) [inline portfolio/src/main/webapp/style.css]: Same for the CSS (2-space indentations)

File: portfolio/src/main/webapp/education.html (hunk 1 above)
Comment (IshaniSanturkar) [reply portfolio/src/main/webapp/education.html]: Hi Hao. To do this, I installed VSCode locally, cloned the repo and made the changes. Should I install VSCode on the Cloud Shell instead?

Comment (fenghaolw) [reply portfolio/src/main/webapp/education.html]: I don't think you can install VScode on cloud shell -- the way you did it is exactly what I mean.

**Review by fenghaolw — APPROVED** [review]

**Pull request merged**